cli/cli/commands/github/login/login.go

```diff
@@ -18,6 +18,7 @@ import (
 
 const (
 	defaultEngineVersion                   = ""
+	defaultEngineAuthor                    = "kurtosistech"
```

Comment:
can we move this to this package: `cli/cli/defaults/defaults.go` ?

Reply:
now I wonder if this value shouldn't be in the launcher and an empty string makes the launcher use the default value. WDYT?